keyrings/alt/tests/test_pyfs.py

```diff
@@ -11,6 +14,18 @@
 from keyring.testing.backend import BackendBasicTests
 from keyring.testing.util import random_string
 
+# support getting module version
+import pkg_resources
```

Comment:
This code looks like a repeat of other code. Probably just use the code from the module.